modules/books.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-globals */
+import addToList from './add.js';
+
+class Books {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('BOOKS', JSON.stringify(this.data));
+    addToList(book);
+  }
```

Comment:
It will be better to make this function an `es6 syntax` function. Remember we need to use `es6` for this milestone. 

**HINT**
```
const timeStamp = () => { your code }
```